### ISSUE TITLE
Expected a single target in []

### ISSUE THREAD

**Issue by zeneodev1** [xavier/xlsx_reader#32]
I was trying to open an excel file but the following error occurred:
```elixir
** (KeyError) key :workbook not found in: {:error, "expected a single target in []"}. If you are using the dot syntax, such as map.field, make sure the left-hand side of the dot is a map
    (xlsx_reader 0.8.4) lib/xlsx_reader/package_loader.ex:134: XlsxReader.PackageLoader.load_styles/2
    (xlsx_reader 0.8.4) lib/xlsx_reader/package_loader.ex:46: XlsxReader.PackageLoader.open/2
```

**Comment by xavier** [comment]
Which software was used to create the file?
Would you be able to provide a sample file which reproduces the issue?

**Comment by zeneodev1** [comment]
After some debugging i found out the file doesn't have shared strings, which makes the following function return an error:
```elixir
  defp load_shared_strings(package) do
    with {:ok, file} <- single_rel_target(package.workbook.rels.shared_strings),
         {:ok, xml} <- extract_xml(package.zip_handle, file),
         {:ok, shared_strings} <- SharedStringsParser.parse(xml) do
      %{package | workbook: %{package.workbook | shared_strings: shared_strings}}
  end
  ```
  
  after making a small modification i could make it work
  ```elixir
  defp load_shared_strings(package) do
    with {:ok, file} <- single_rel_target(package.workbook.rels.shared_strings),
         {:ok, xml} <- extract_xml(package.zip_handle, file),
         {:ok, shared_strings} <- SharedStringsParser.parse(xml) do
      %{package | workbook: %{package.workbook | shared_strings: shared_strings}}
    else
      {:error, "expected a single target in []"} -> package
      any -> any
    end
  end
  ```

**Comment by xavier** [comment]
👍 Thanks for your report, the issue should be fixed in 0.8.5!
